test/core/test/lot-of-tests.test.ts

```diff
@@ -0,0 +1,13 @@
+import { describe, expect, it } from 'vitest'
+
+describe('Suite of 500 tests for UI performance tests', () => {
+  for (let index_d = 1; index_d <= 50; index_d++) {
```

Comment:
Can we have the magic numbers as const of the top level so we can modify them easier?